core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -32,11 +32,26 @@ oppia.factory('UrlInterpolationService', [
       }
     };
 
-    var getCachePrefixedUrl = function(resourcePath) {
+    var getPrefixedUrl = function(resourcePath) {
       validateResourcePath(resourcePath);
       return GLOBALS.ASSET_DIR_PREFIX + resourcePath;
     };
 
+    var getUrlWithSlug = function(resourcePath) {
```

Comment:
In order to test these 'private' functions, I would need to return them (like I did for _getExtensionResourceUrl_) is this okay? (I would prefix them with '_')